src/selectors/index.js

```diff
@@ -5,3 +5,7 @@ export function sectionsSelector(state) {
 export function currentBrowseLevelSelector(state) {
   return state.getIn(['browse', 'currentBrowseLevel']);
 }
+
+export function isFetchingSelector(state) {
```

Comment:
This is shared between all browse level container components. Eventually we'll do a ticket to move all selectors into the `selectors` directory for discoverability and shared usage.